Formula/chezscheme.rb

```diff
@@ -0,0 +1,27 @@
+class Chezscheme < Formula
+  desc "Chez Scheme"
+  homepage "https://cisco.github.io/ChezScheme/"
+  url "https://github.com/cisco/ChezScheme/archive/v9.4.tar.gz"
+  sha256 "9f4e6fe737300878c3c9ca6ed09ed97fc2edbf40e4cf37bd61f48a27f5adf952"
+
+  depends_on :x11 => build
```

Comment:
`build` isn't a valid tag. It should be `:build` if necessary, but note that if it uses X11 we'd want this to be filed in the `homebrew/x11` [tap](https://github.com/Homebrew/homebrew-x11).